posts/2021/09/low-code-workshop.md

```diff
@@ -0,0 +1,182 @@
+
+To kick off the work, the core developer walks us through the pull request
+they've prepared and introduces different files.  For reinforcement, the
+facilitator will share their screen to demonstrate the process of recommending
+changes to files in a pull request.
+
+### Group work
+
+Together the new and old contributors work together to make changes to relevant
+files to improve their quality.  These changes are made using GitHub's suggest
+changes feature that allows authors to make single line changes to code; a
+workflow perfect for alt text.
+
+### Group review
+
+Over 40 minutes several contributors can make quite a few commits.  The project
+contributor shares their screen to review the suggestions made over the course
+of the event.  This portion becomes a critique to review the different
+contributions made.
+
+### Bringing everything home
+
+All of the work we talked about happened on a contributor's fork.  They now
+have a pull request withl a gaggle of contributors.  Their last job is to
+submit the pull request to the mainline project and see the work accepted into
+the code base.
+
+## Success stories
+
+Everyone loves an underdog story.  What follows are there triumphs of some open
+source allies who thought they could run an hour sprint that allowed multiple
+folks to contribute to an open source project.  Believe it or not, we weren't
+lying to ourselves and others. Below we highlight our successes with running
+low-code sprints in Jupyter and NumPy.
+
+### Jupyter accessibility group
+
+There is a small group of the Jupyter community focused on the improving the
+accessibility of JupyterLab and orbiting projects.  We meet every other work to
+add small patches and fixes. Currently, we're in the process of organizing
+workshops to advocate for accessibility and improve the quality of the entire
+Jupyter experience for disabled scientists.
+
+On an off week, between our normal syncs, we tested a low-code format for folks
+to participate in the Jupyter project.  During an hour, four people were
+included in the project by suggesting changes to alt text for images in the
+JupyterLab documentation (https://github.com/isabela-pf/jupyterlab/pull/1),
+resulting in a collaborative commit to mainline with the help of @krassowski
+and @isabela-pf as champions for the shared
+work.(https://github.com/jupyterlab/jupyterlab/pull/10670)
+
+We consider this event a success because we were able to remove `git` as a
+technical barrier to entry and support changes from multiple authors.
+
+
+### [NumPy documentation](https://numpy.org/doc/stable/) new comers meeting
+
+Our fledgling experience in the Jupyter event made us *sort of* confident this
+could work for other projects.  So we contacted our NumPy accessibility allies
+@melissawm and @marsbarlee to test this hypothesis at the NumPy newcomer's
+meetings, which [happen every other
+Thursday](http://numpy-discussion.10968.n7.nabble.com/Newcomer-s-Hour-td49226.html).
```

Comment:
This link is broken, and I'm not sure what it was pointing to. Can you please find the relevant post on https://mail.python.org/archives/list/numpy-discussion@python.org/ @melissawm?

Reply:
```suggestion
Thursday](https://mail.python.org/archives/list/numpy-discussion@python.org/thread/4EBH3QAH6IR56WJCM7VEL55ACGGK6JKP/).
```